espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/handlers/AcceptAlert.java

```diff
@@ -42,7 +43,7 @@ public Void handle(AppiumParams params) throws AppiumException {
                 .getInstance(InstrumentationRegistry.getInstrumentation());
         final List<UiObject2> dialogs = mDevice.findObjects(By.clazz(Dialog.class));
         if (dialogs.isEmpty()) {
-            throw new AppiumException("No alerts can be detected on the screen");
+            throw new NoAlertOpenException("No alerts can be detected on the screen");
```

Comment:
```"No alerts can be detected on the screen"``` string can be moved to the  ```NoAlertOpenException``` class and used in the default constructor.

Reply:
Changed